ci-operator/step-registry/provider-certification-tool/run/provider-certification-tool-run-commands.sh

```diff
@@ -20,3 +20,50 @@ mkdir -p "${ARTIFACT_DIR}/certification-results"
 
 # Run results summary (to log to file)
 ./openshift-provider-cert-linux-amd64 results "${ARTIFACT_DIR}"/certification-results/*.tar.gz
+
+#
+# Gather some cluster information and upload certification results
+#
+
+export AWS_DEFAULT_REGION=us-west-2
+export AWS_SHARED_CREDENTIALS_FILE=/var/run/vault/opct/.awscred
+export AWS_MAX_ATTEMPTS=50
+export AWS_RETRY_MODE=adaptive
+export HOME=/tmp
+
+# Install AWS CLI
+if ! command -v aws &> /dev/null
+then
+    echo "$(date -u --rfc-3339=seconds) - Install AWS cli..."
+    export PATH="${HOME}/.local/bin:${PATH}"
+    if command -v pip3 &> /dev/null
+    then
+        pip3 install --user awscli
+    else
+        if [ "$(python -c 'import sys;print(sys.version_info.major)')" -eq 2 ]
+        then
+          easy_install --user 'pip<21'
+          pip install --user awscli
+        else
+          echo "$(date -u --rfc-3339=seconds) - No pip available exiting..."
+          exit 1
+        fi
+    fi
+fi
+
+# install newest oc
+export PATH=$PATH:/tmp/bin
+mkdir /tmp/bin
+curl -L --fail https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz | tar xvzf - -C /tmp/bin/ oc
+chmod ug+x /tmp/bin/oc
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+VERSION=$(oc get clusterversion version -o=jsonpath='{.status.desired.version}')
+
+# upload to AWS S3
+DATE=$(date +%Y%m%d)
+
+echo "s3://openshift-provider-certification/baseline-results/${VERSION}-${DATE}.tar.gz"
+echo '{"platform-type":"'"${CLUSTER_TYPE}"'"}'
+
+aws s3 cp "${ARTIFACT_DIR}"/certification-results/*.tar.gz "s3://openshift-provider-certification/baseline-results/${VERSION}-${DATE}.tar.gz" \
```

Comment:
I am still uncertain if we need to add the baseline platform on the name of the results - as it is external, I can, for example, run on AWS and upload to the bucket the results. But metadata for now sgtm.

Reply:
Sounds good to me as well. I think as long as we don't heavily depend on the metadata somehow we can change that up later as requirements change.